source/open_cluster_deployment/common_node/apparmor.txt

```diff
@@ -1,7 +1,7 @@
 Depending on the type of OpenNebula deployment, the AppArmor can block some operations initiated by the OpenNebula Front-end, which results in a failure of the particular operation.  It's **not recommended to disable** the apparmor on production environments, as it degrades the security of your server, but to investigate and workaround each individual problem, a good starting point is `AppArmor HowToUse Guide <https://wiki.debian.org/AppArmor/HowToUse/>`__. The administrator might disable the AppArmor to temporarily workaround the problem or on non-production deployments the steps for disabling it can be found `here <https://wiki.debian.org/AppArmor/HowToUse#Disable_AppArmor>`__.
 
-.. note:: Depending on your OpenNebula deployment type, the following lines might be required at ``/etc/apparmor.d/abstractions/libvirt-qemu`` profile:
+.. note:: Depending on your OpenNebula deployment type, the following lines might be required at ``/etc/apparmor.d/local/abstractions/libvirt-qemu`` profile:
```

Comment:
The patch seems to be wrong, in 2404:
```
$ cat /etc/os-release
PRETTY_NAME="Ubuntu 24.04 LTS"
NAME="Ubuntu"
VERSION_ID="24.04"
VERSION="24.04 LTS (Noble Numbat)"
VERSION_CODENAME=noble
```

The path in the documentation seems to be right (i.e. no local) ?
```
$ ls -la /etc/apparmor.d/abstractions/libvirt-qemu
-rw-r--r-- 1 root root 9776 Jul  1 18:12 /etc/apparmor.d/abstractions/libvirt-qemu
```